### PR TITLE
Feat/match tiles

### PR DESCRIPTION
**Pull request by louvrecly**
### Descriptions
- Implement the `MatchTiles` component in the UI

### Related
- https://github.com/betterbettor/bet-better-react/issues/11

### Stories
- [x] create all necessary interfaces as defined in the backend server
- [x] configure `images` property with `remotePatterns` option in the `nextConfig` to whitelist images data from the `api-sports.io` domain
- [x] create basic UI structure on the `Home` page
- [x] extract `MatchTiles` and `MatchTile` components from the `Home` page UI

### Screenshots
|Before|After|
|-|-|
| image | image |

### CODE REVIEW

File: interfaces/match.interface.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import { OddsValues } from './odds.interface';
+import Team from './team.interface';
+
+interface Match {
+  id: number;
+  startTime: number;
+  league: League;
+  home: Team;
+  away: Team;
+  ttl: number;
```
Comment (krissolui) [inline interfaces/match.interface.ts:11]: `ttl` has been removed from api response.
```suggestion
```

File: app/components/match-tiles/match-tile.tsx
Comment (krissolui) [inline app/components/match-tiles/match-tile.tsx:1]: 3 div for displaying Home, Away and Draw value can be refactored to be a separated component.

**Review by krissolui — COMMENTED** [review]
Made a few minor comments. Rest are fine

**Review by wkip1220withweb — APPROVED** [review]

File: app/components/match-tiles/match-tile.tsx
Comment (charrrleee) [reply app/components/match-tiles/match-tile.tsx:1]: I agree

**Review by charrrleee — APPROVED** [review]

**Review by krissolui — APPROVED** [review]

File: app/components/match-tiles/match-tile.tsx
Comment (louvrecly) [reply app/components/match-tiles/match-tile.tsx:1]: Agreed. Will create another PR for the refactoring of components.

File: interfaces/match.interface.ts (hunk 1 above)
Comment (louvrecly) [reply interfaces/match.interface.ts]: removed `ttl` [here](https://github.com/betterbettor/bet-better-react/pull/13/commits/9ecd344c04abf56eb32f28f9fd41947d61a60377#diff-f1b72785c1717d407cf939f9080b9049d4243ceb503720ba2fa51ee4c5b24809L11) as suggested!

**Review by DivyaJain9 — COMMENTED** [review]
Looks Good to me

**Pull request merged**